src/memex/models/document.py

```diff
@@ -409,6 +405,34 @@ def merge_documents(session, documents, updated=None):
     return master
 
 
+def best_document_uri_from(document_uris):
```

Comment:
I'm not sure `best` is particularly helpful without qualification. Best for what? Identity? Location?